platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -216,27 +192,29 @@ private void writeOutUsersDotAttributesFile(File userAttributesFile) {
     Map<String, Object> json = null;
     try (InputStream stream = Files.newInputStream(Paths.get(userAttributesFile.toURI()))) {
       json = MAPPER.parser().parseMap(stream);
+    } catch (IOException e) {
+      LOGGER.warn("Unable to read system user attribute file for hostname update.", e);
```

Comment:
✏️  Yet we fall-through to log that Json was not found ...
❓  Should we not just return right away?

Reply:
good catch